app/views/state_file/questions/primary_state_id/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if current_state_code == "az" %>
+  <% render 'state_file/questions/az_primary_state_id/az_primary' %>
+<% elsif current_state_code == "md" %>
+  <% render 'state_file/questions/md_primary_state_id/md_primary' %>
+<% elsif current_state_code == "nc" %>
+  <% render 'state_file/questions/nc_primary_state_id/nc_primary' %>
+<% end %>
```

Comment:
[pebble -- avoiding state-code dependent conditionals] could we deprecate this file by assigning the partial/view to render in the controller action?